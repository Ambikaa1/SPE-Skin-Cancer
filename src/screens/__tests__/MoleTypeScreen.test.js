// import React from 'react';
// import renderer from 'react-test-renderer';
//
// import MoleTypeScreen from "../MoleTypeScreen";
//
// describe('<MoleTypeScreen />', () => {
//     it('renders correctly', () => {
//         const tree = renderer.create(<MoleTypeScreen />).toJSON();
//         expect(tree).toMatchSnapshot();
//     });
// });
