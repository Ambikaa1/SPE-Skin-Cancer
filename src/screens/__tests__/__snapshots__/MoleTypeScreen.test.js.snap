// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoleTypeScreen /> renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "marginLeft": 10,
      "marginTop": 10,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
      }
    }
  >
    Is this a new mole?
  </Text>
  <View
    style={
      Array [
        Object {
          "height": 40,
          "marginRight": 10,
          "marginTop": 5,
        },
        Object {
          "zIndex": 5000,
        },
      ]
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onLayout={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "borderBottomLeftRadius": 5,
          "borderBottomRightRadius": 5,
          "borderColor": "#dfdfdf",
          "borderTopLeftRadius": 5,
          "borderTopRightRadius": 5,
          "borderWidth": 1,
          "flex": 1,
          "flexDirection": "row",
          "opacity": 1,
          "paddingHorizontal": 10,
          "paddingVertical": 5,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "fontSize": 20,
            },
            Object {},
            Object {
              "flex": 1,
              "opacity": 1,
            },
            Object {
              "marginLeft": 5,
              "marginRight": 0,
            },
            false,
            Object {
              "marginLeft": 5,
            },
          ]
        }
      >
        Select an item
      </Text>
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderBottomLeftRadius": 0,
              "borderTopLeftRadius": 0,
              "flexDirection": "row",
              "justifyContent": "center",
              "paddingVertical": 8,
            },
          ]
        }
      >
        <View
          style={
            Array [
              Object {},
              Object {
                "opacity": 1,
              },
            ]
          }
        >
          <Text />
        </View>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#fff",
            "borderBottomLeftRadius": 5,
            "borderBottomRightRadius": 5,
            "borderColor": "#dfdfdf",
            "borderTopLeftRadius": 5,
            "borderTopRightRadius": 5,
            "borderWidth": 1,
            "paddingHorizontal": 10,
            "paddingVertical": 5,
          },
          Object {
            "alignItems": "center",
            "borderTopLeftRadius": 0,
            "borderTopRightRadius": 0,
            "justifyContent": "center",
            "position": "absolute",
            "width": "100%",
          },
          Object {},
          Object {
            "borderWidth": 0,
            "display": "none",
            "position": "relative",
          },
          Object {
            "maxHeight": 150,
            "top": undefined,
            "zIndex": 5000,
          },
        ]
      }
    >
      <RCTScrollView
        nestedScrollEnabled={true}
        style={
          Object {
            "width": "100%",
          }
        }
      >
        <View>
          <View
            onLayout={[Function]}
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingVertical": 8,
                  "width": "100%",
                }
              }
            >
              <View
                style={
                  Object {
                    "alignContent": "center",
                    "flexDirection": "row",
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "fontSize": 20,
                      },
                      false,
                      Object {},
                    ]
                  }
                >
                  Yes
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingVertical": 8,
                  "width": "100%",
                }
              }
            >
              <View
                style={
                  Object {
                    "alignContent": "center",
                    "flexDirection": "row",
                  }
                }
              >
                <Text
                  style={
                    Array [
                      Object {
                        "fontSize": 20,
                      },
                      false,
                      Object {},
                    ]
                  }
                >
                  No
                </Text>
              </View>
            </View>
          </View>
        </View>
      </RCTScrollView>
    </View>
  </View>
</View>
`;
